src/codewhisperer/util/supplementalContext/supplementalContextUtil.ts

```diff
@@ -0,0 +1,78 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { fetchSupplementalContextForTest } from './utgUtils'
+import { fetchSupplementalContextForSrc } from './crossFileContextUtil'
+import { isTestFile } from './codeParsingUtil'
+import { DependencyGraphFactory } from '../dependencyGraph/dependencyGraphFactory'
+import * as vscode from 'vscode'
+import * as codewhispererClient from '../../client/codewhisperer'
+import { CancellationError } from '../../../shared/utilities/timeoutUtils'
+import { ToolkitError } from '../../../shared/errors'
+import { getLogger } from '../../../shared/logger/logger'
+
+const performance = globalThis.performance ?? require('perf_hooks').performance
+
+export interface CodeWhispererSupplementalContext {
+    isUtg: boolean
+    isProcessTimeout: boolean
+    contents: codewhispererClient.SupplementalContext[]
+    contentsLength: number
+    latency: number
+}
+
+export async function fetchSupplementalContext(
```

Comment:
No docstring in this file. Same for other non-trivial files.

Reply:
Is this a blocker for this release? wondering can I make it as a followup

Reply:
not a blocker